pycbc/psd/read.py

```diff
@@ -48,7 +48,7 @@ def from_numpy_arrays(freq_data, noise_data, length, delta_f, low_freq_cutoff):
         raise ValueError('Lowest frequency in input data '
           ' is higher than requested low-frequency cutoff ' + str(low_freq_cutoff))
 
-    kmin = int(low_freq_cutoff / delta_f)
+    kmin = int(numpy.ceil(low_freq_cutoff / delta_f))
```

Comment:
Thinking about this a bit more, perhaps there is an easier solution that would be in general nicer anyway. Would it make sense to allow the code to extrapolate up to a sample away from the data set at the boundaries?

Reply:
@ahnitz I have considered this, I will test just fill `np.inf` first.